packages/next-server/server/render.tsx

```diff
@@ -344,7 +344,16 @@ export async function renderToHTML(
   )
 
   try {
-    props = await loadGetInitialProps(App, { Component, router, ctx })
+    props = await loadGetInitialProps(App, {
+      Component,
+      AppTree: (props: any) => (
+        <AppContainer>
+          <App {...props} />
```

Comment:
Why does the other reference need `<App Component={Component} router={router} {...props} />` but this one only need `props`? Seems like a bug.

Reply:
updated